test/Locker.t.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import {Test} from "forge-std/Test.sol";
+import {Locker} from "../src/libraries/Locker.sol";
+
+contract LockerTest is Test {
+    address constant ADDRESS_AS = 0xaAaAaAaaAaAaAaaAaAAAAAAAAaaaAaAaAaaAaaAa;
+    address constant ADDRESS_BS = 0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB;
+
+    function test_setLocker(address locker) public {
```

Comment:
for fuzz test I would add the _fuzz prefix.

so its test_fuzz

good way to standardize and be able to use regex to filter out tests if we need in CI or anywhere else